### PR TITLE
feat(admin-ui): init components context page

### PR DESCRIPTION
**Pull request by frdomovic**
# feat(admin-ui): components for context page
 
## Summary:
This PR includes initial build components for context page. 
- Includes flex wrappers and container for the page.
- list components including table row item, table header, table switch context (content).
- form for starting context TBD

Fixes # C20-60

## Test plan:
---Starting context

Screenshot 2024-05-08 at 15 35 21

---Components

TITLE + BUTTON OPTION + DESCRIPTION + LIST
Screenshot 2024-05-03 at 16 01 29

OPTION HEADER
Screenshot 2024-05-03 at 16 01 34

LIST WITH NO SWITCH CONTEXT 
Screenshot 2024-05-03 at 16 01 49

COMBINED
Screenshot 2024-05-03 at 16 01 58

### CODE REVIEW

File: node-ui/src/api/dataSource/ContextDataSource.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        console.error("Error fetching contexts:", error);
+        return [];
+      }
+    }
+
+    async startContexts(applicationId, initArguments) {
+        const argumentsStringify = JSON.stringify(initArguments);
+
+        try {
+            const response = await this.client.post("/admin-api/context", {applicationId, argumentsStringify});
```
Comment (frdomovic) [inline node-ui/src/api/dataSource/ContextDataSource.js]: @miraclx what else do we expect from this api call for starting context? 
Will update the body sent with proper arguments but in general, applicationId (checking if blob is saved locally) + initialisation arguments for the application

Comment (miraclx) [reply node-ui/src/api/dataSource/ContextDataSource.js]: appId, and a conditional pair of (init method + args) is all

File: node-ui/src/components/common/Button.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  line-height: 1.25rem;
+  text-align: center;
+  cursor: pointer;
+
+  &:hover {
+    background-color: #76f5f9;
+  }
+`;
+
+export default function Button({ onClick, text }) {
```
Comment (frdomovic) [inline node-ui/src/components/common/Button.tsx]: This component is going to be build more with features as updated. Will cleanup components after switched to new design

File: node-ui/src/components/common/OptionsHeader.jsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  setCurrentOption,
+  showOptionsCount,
+}) {
+  return (
+    <HeaderGrid optionsCount={tableOptions?.length ?? 0}>
+      {tableOptions.map((option, index) => {
+        return (
+          <div
+            className={`header-option ${
+              currentOption === option.name ? "active" : ""
```
Comment (frdomovic) [inline node-ui/src/components/common/OptionsHeader.jsx]:
```suggestion
              currentOption === option.id ? "active" : ""
```

File: node-ui/package.json [hunk 4]
```diff
@@ -33,7 +33,7 @@
     "react-dom": "^18.2.0",
     "react-router-dom": "^6.22.3",
     "react-tooltip": "^5.26.3",
-    "styled-components": "^6.1.8"
+    "styled-components": "^5.3.11"
```
Comment (MatejVukosav) [inline node-ui/package.json]: Why did u degrade version?

Comment (frdomovic) [reply node-ui/package.json]: This issue with using react and styled component props -> as you cant define component props
https://stackoverflow.com/questions/57586654/styled-component-attrs-react-does-not-recognize-prop -> haven't seen this fix yet but will try it.

Comment (frdomovic) [reply node-ui/package.json]: fixed

**Pull request merged**